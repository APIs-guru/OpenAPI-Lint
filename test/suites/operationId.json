{
  "title": "multiple operations with the same operationId",
  "patch": {
    "paths": {
      "/pet": {
        "put": {
          "operationId": "addPet"
        }
      }
    }
  }
}
