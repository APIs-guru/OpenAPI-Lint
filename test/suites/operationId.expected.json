{
  "DUPLICATE_OPERATIONID": [
    {
      "code": "DUPLICATE_OPERATIONID",
      "params": [
        "addPet"
      ],
      "type": "error",
      "message": "Cannot have multiple operations with the same operationId: addPet",
      "path": [
        "paths",
        "/pet",
        "post",
        "operationId"
      ]
    }
  ]
}